### DISCUSSION TITLE
Projects should either be org-wide or entirely repo based.

### DISCUSSION THREAD

**Discussion by c0dezer019** [community/community#20953] [Projects and Issues]
# The problem
Currently, if a repo is part of an org you have to go to the main org view to create an org-wide project. Then you have to go to a repo and 'attach' it to the repo, of which then you can add issues to the project. However, this means that you have org projects unrelated to other repo visible to all, and maybe you don't want repo-specific projects being visible to the entire org.

Also, this could lead to difficulty adding issues and PRs to a project. Imagine an org having several different repos in the teens. Obviously now this means that you could have a multitude of issues with \#2. This could lead to accidentally linking the wrong issue, which could lead to a disaster if the org mandates that developers refer to the Projects tab to complete tasks.

# The Solution
Projects need to have an additional setting to be set on and after creation, giving the ability for it to be directly tied to a repo, hiding it from Org Project Tab, or making it an Org-wide project. They could be tied to specified repos if multiple repos rely on one another and a project is set to achieve a certain goal involving all involved repos. This would make currently loose project/repo relationship a bit tighter.

To address the issue search problem, the abilty to do something like "[project] #issueNumber" would make searching for specific issues a tad bit easier.

**Comment by eallenOP** [comment]
Yes, this is the main problem my teams have with the new Projects Beta and why we were sticking with Projects Classic. But now Classic seems to have been removed from new repos! Why force people to use a Beta?? Anyway, please make Projects able to be repo-based only.

**Comment by EricPedley** [comment]
Another possibly easier solution would be adding links to all repos that are linked to a project board on the page to view it. I find it annoying when I look at our project board and then want to open the repo in a new tab, but there is no button to do so.

**Comment by InsertCreativityHere** [comment]
I agree, my organization has recently made the switch, and so far it seems fine!

But we have around 40 different repositories, and some of them had their own (classic) projects which existed solely for a specific repo. Now these are all org-wide projects, which feels kind of cluttered and mixed together. I think having projects that can span over multiple repos is great in some cases, but tying a project to a single repo (not at the org level) is a necessary option IMO.
